.github/workflows/unix_mac.yml

```
@@ -34,7 +34,7 @@ jobs:
     runs-on: macos-10.15
```

Unless there are other failures, this should be enough to fix the mac builds
```suggestion
    runs-on: macos-11
```